bindings/cs/MongoDB.Crypt/Algorithm.cs

```diff
@@ -18,7 +18,7 @@ namespace MongoDB.Crypt
 {
     public enum Alogrithm 
     {
-        AEAD_AES_256_CBC_HMAC_SHA_512_Randomized = 0,
+        AEAD_AES_256_CBC_HMAC_SHA_512_Random = 0,
```

Comment:
I think we used `Randomized` everywhere else.  Why change it?

Reply:
Hm, I thought so too, but it seems `Random`  is what's being used in the C code [here ](https://github.com/vincentkam/libmongocrypt/blob/csharp/src/mongocrypt-ctx.c#L24 ). The CS binding [ unit test ](https://github.com/vincentkam/libmongocrypt/blob/csharp/bindings/cs/MongoDB.Crypt.Test/BasicTests.cs#L200)seems to pass with the value of `AEAD_AES_256_CBC_HMAC_SHA_512-Random` as well fwiw

Reply:
Indeed.  Random is also used in the spec tests.